test/driver.js

```diff
@@ -582,6 +582,16 @@ var Driver = (function DriverClosure() { // eslint-disable-line no-unused-vars
       this._log('Done !');
       this.end.textContent = 'Tests finished. Close this window!';
 
+      var re = new XMLHttpRequest();
+      re.open('POST','/browserTestReports?path=' + escape(this.appPath),false);
+      re.onreadystatechange = function () {
+        if (re.readyState == 4) {
+          var browserTestInfo = JSON.stringify(window.__coverage__);
+          re.send(browserTestInfo);
+        }
+      }
```

Comment:
This code is not doing what you are intending. I suggest to read the tutorial and documentation at MDN to understand what the API is doing and how you should be using it instead:

- https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest
- https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/Using_XMLHttpRequest

Reply:
on it :+1: